data_validation/combiner.py

```diff
@@ -210,7 +209,11 @@ def _calculate_differences(
 
 def _pivot_result(result, join_on_fields, validations, result_type):
     all_fields = frozenset(result.schema().names)
-    validation_fields = all_fields - frozenset(join_on_fields)
+    validation_fields = (
+        all_fields - frozenset(join_on_fields)
+        if "hash__all" not in join_on_fields
```

Comment:
Should we check if custom_query_type = row field instead of if hash_all not in join_on_fields

Reply:
So I am actually setting this specifically in the dat_validation function. For rest all other conditions I am using if custom_query = row/column check